input/CIB_Lenz.yml

```diff
@@ -0,0 +1,17 @@
+mapper_class: MapperCIBLenz
+map_name: CIB_Lenz
+file_map: /mnt/extraspace/damonge/Datasets/CIB_Lenz/cib_fullmission.hpx.fits
+file_hm1: /mnt/extraspace/damonge/Datasets/CIB_Lenz/cib_evenring.hpx.fits
+file_hm2: /mnt/extraspace/damonge/Datasets/CIB_Lenz/cib_oddring.hpx.fits
+file_mask: /mnt/extraspace/damonge/Datasets/CIB_Lenz/mask_apod.hpx.fits
+gal_mask_mode: '0.2'
+sp_mask_mode:  '545'
+beam_info: [{'type': 'Custom', 'file': '/mnt/extraspace/damonge/Datasets/CIB_Lenz/windowfunctions.csv',
+             'field': 'Wl_eff'},
+             {'type': 'Pixel',
+             'nside_native': 2048,
+             'nside_wanted': 4096}]
+coords: G
+custom_auto: True
+mask_name: mask_CIB_Lenz 
```

Comment:
note that the mask name will be different for different frequencies

Reply:
Done!